tests/test_ble_lifecycle_receive_targets.py

```
@@ -67,14 +67,22 @@ def _legacy_cleanup(func: Any, _operation_name: str) -> None:
         "cleanup-op",
     )
 
-    def _keyword_incompatible_safe_execute(func: Any, *args: object, **kwargs: object) -> object:
+    def _keyword_incompatible_safe_execute(
+        func: Any, *args: object, **kwargs: object
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `func`

(ANN401)